### PR TITLE
Feature/application portal UI refine -- submit response and disabled nav styles

### PR DESCRIPTION
**Pull request by Alperdec**
Problem
=======
disabled application nav buttons and application submit response required styling

Some spelling errors and special symbols in Countries.js

Solution
========
Added styles to submit response and styled disabled application nav buttons.
Fixed spelling errors

Change Summary:
---------------
changes to
`src/views/Portal/components/ApplicationForm/index.scss`
 `src/views/Portal/components/ApplicationForm/index.view.tsx`
   `src/views/Portal/utils/Countries.js`

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
It's pretty minimal

### CODE REVIEW

File: src/views/Portal/components/ApplicationForm/index.view.tsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ const ApplicationForm: React.FC = () => {
         <div className='application-form-component__response'>
           {successOnSubmit === "submitted" ? "submitted" : ""}
         </div>
-        <div className='application-form-component__response'>
+        <div className='application-form-component__response-err'>
           {successOnSubmit === "error submitting" ? "Could Not Submit" : ""}
```
Comment (krishnakpandian) [inline src/views/Portal/components/ApplicationForm/index.view.tsx]: Can we update the message to be: "CruzHacks Cloud had an error processing your application. There may be a high bandwidth of users at this moment. Our engineers have been alerted! Try again soon!"

**Review by krishnakpandian — CHANGES_REQUESTED** [review]

**Review by w-gao — APPROVED** [review]
Other than Krishna's comment, this looks pretty good to me!

File: src/views/Portal/components/ApplicationForm/index.view.tsx (hunk 1 above)
Comment (Alperdec) [reply src/views/Portal/components/ApplicationForm/index.view.tsx]: Sure! will do

**Review by krishnakpandian — APPROVED** [review]
Looks good! Let's update the PR template before merging so we can have reference to this in the future on this changes and why we made them.

**Pull request merged**